build.gradle

```diff
@@ -112,16 +114,21 @@ def afuChild      = fromBaseDir.curry(afuDir)
 def plumeDir        = env["PLUME"] ?: jsr308Child("plume-lib")
 def plumeChild      = fromBaseDir.curry(plumeDir)
 
+def langtoolsChild = fromBaseDir.curry(jsr308_langtools)
+
 //JarsToPackage contains both all members that should be on the classpath for this build
 //and all jars (except for checker-framework-inference.jar) that should be included in
 //the jar produced by the allJars command
 def jarsToPackage = [ //Annotation File Utilities paths        //TODO: AFU Jar has some things that are also in checkers.jar
                         "annotation-file-utilities.jar",
                     ].collect { afuChild(it).getAbsolutePath()      } +
+                    [
+                        "dist/lib/javac.jar",
+                    ].collect { langtoolsChild(it).getAbsolutePath() } +
 
                     [//checker relative paths
-                        "checker/dist/$jreJarName",
-                        "checker/dist/javac.jar",
+                        //"checker/dist/$jreJarName",
```

Comment:
A comment why this is commented out would be good.
Currently none of the checkers uses the annotated JDK, so we don't require it. However, this will result in a warning when running a checker, right?